src/Spd.Manager.Licence/BizLicAppContract.cs

```diff
@@ -77,6 +77,7 @@ public record BizLicAppResponse : BizLicenceApp
 
     // Contains branding, insurance, registrar, security dog certificate and BC report documents
     public IEnumerable<Document>? DocumentInfos { get; set; }
+    public ContactInfo? BizManagerContactInfo { get; set; }
```

Comment:
If we remove it from application, why do we still need to return it? UI does not need it anymore.

Reply:
I left it here because that field is not defined in the related Business Profile GET endpoint. I can move it to that service if you are ok with that.

Reply:
yes, you need to move it to business profile response and request.

Reply:
Fixed